### ISSUE TITLE
[React Native Gallery -> Pressable]: Action is defined for disabled button i.e., for 'Disabled Pressable' button present under "A disabled Pressable component".

### ISSUE THREAD

**Issue by YajurG** [microsoft/react-native-gallery#363]
**Repro-Steps:**

- Launch the React Native Gallery app.
- Navigate to ‘Pressable’ button and invoke it.
- Navigate to 'Disabled Pressable' button, invoke it and observe.

**Actual Result:**

Action is defined for disabled button. On invoking 'Disabled Pressable' button action is being performed, which is incorrect.

**Expected Result:**

For disabled buttons action should be defined. When user navigate and invoke 'Disabled Pressable' button, no action should be performed. 

**User Impact:**

Users with motor disabilities who rely on keyboard will be impacted and misguided if some action is defined for disabled buttons.

Internal Bug Link: https://dev.azure.com/microsoft/OS/_workitems/edit/47843510
